test/unitary/bin/cli-runner.test.ts

```diff
@@ -11,6 +11,7 @@ import { defaultFlowResult, FlowResult } from "@bc/domain/flow";
 import { ExecuteCommandResult, ExecutionResult } from "@bc/domain/execute-command-result";
 import { UploadResponse } from "@actions/artifact";
 import { ExecuteNodeResult } from "@bc/domain/execute-node-result";
+import { defaultNodeValue } from "@bc/domain/node";
```

Comment:
as I mentioned before, this should be moved to configuration-reader library, right?

Reply:
IMO the default value is something project specific and should not be a part of the configuration-reader. Over here we are using the default value as 
```
export const defaultNodeValue: Readonly<Node> = {
  project: "",
  parents: [],
  children: []
};
```
but maybe for some project we might need it as 
```
export const defaultNodeValue: Readonly<Node> = {
  project: "kiegroup/drools",
  parents: [],
  children: []
};